### PR TITLE
fix: Rename "Technical Guide" category to "app.opensauced.pizza"

### LINKED ISSUE #50: docs: Rename "Technical Guide" category to "app.opensauced.pizza"
### Type of feature

📝 Documentation

### Current behavior

Since the docs.opensauced is supposed to be the official documentation for all open-sauce repos, having a category with a very general name like "Technical Guide" is very confusing, especially that it focuses mostly on just app.opensauce, without even referencing it.
image

### Suggested solution

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by Vinitvh**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

Renamed “Technical Guide” category to "app.opensauced.pizza"
## Related Tickets & Documents

fix #50 
## Mobile & Desktop Screenshots/Recordings

opensauce

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by Deadreyo** [comment]
Hello @Vinitvh 

Please rename the folder `./docs/technical/` to `./docs/AppOpensauce` as well

File: sidebars.js [hunk 1]
```diff
@@ -24,13 +24,13 @@ module.exports = {
     },
     {
       type: 'category',
-      label: 'Technical guide',
+      label: 'app.opensauced.pizza',
```
Comment (0-vortex) [inline sidebars.js:27]: just the label is fine, the items don't have to actually be moved around :D

**Review by 0-vortex — CHANGES_REQUESTED** [review]
Changing the label is enough, if you want to also change folder structure that is fine as long as you keep naming to lowercase, and do the rename :D 

In the case of this PR, you have added those files as a new category, keeping the code for the old ones :D It should be one or the other

**Comment by bdougie** [comment]
With this change, I would recommend `resolve-merge-conflicts` be moved to the Contributing guide section, since it is not app.opensauced.pizza specific.

File: sidebars.js (hunk 1 above)
Comment (Vinitvh) [reply sidebars.js:27]: @0-vortex Sorry I didn't get you. You mean, should I change label to `app.opensauced.pizza` and keep folder name as `technical` as it is, right?

File: sidebars.js [hunk 2]
```diff
@@ -20,17 +20,24 @@ module.exports = {
...
+<<<<<<< HEAD
+        'AppOpensauce/introduction-to-storybook',
+        'AppOpensauce/dark-mode-explained',
+        'AppOpensauce/fetching-data-from-github',
+        'AppOpensauce/resolve-merge-conflicts',
+=======
+        'appopensauce/introduction-to-storybook',
+        'appopensauce/dark-mode-explained',
+        'appopensauce/fetching-data-from-github',
+>>>>>>> 012a579 (fix: Move resolve-merge-conflicts to contributing and rename Appopensauce)
```
Comment (bdougie) [inline sidebars.js:40]: Looks like some merge conflicts snuck in here. Can you clean this up?

**Review by bdougie — CHANGES_REQUESTED** [review]
Instead of `appopensauce` it should be `appOpensauced` - note the `d` at the end.

File: sidebars.js (hunk 1 above)
Comment (0-vortex) [reply sidebars.js:28]: yes, at the moment the folder is being added 3 times to the PR

**Review by bdougie — APPROVED** [review]

**Comment by bdougie** [comment]
moving in a different direction for the docs.

**Pull request closed without merging**